service/src/main/java/bio/terra/user/service/iam/SamService.java

```diff
@@ -0,0 +1,53 @@
+package bio.terra.user.service.iam;
+
+import bio.terra.common.exception.ForbiddenException;
+import bio.terra.common.iam.BearerToken;
+import bio.terra.common.sam.SamRetry;
+import bio.terra.user.app.configuration.SamConfiguration;
+import org.broadinstitute.dsde.workbench.client.sam.ApiClient;
+import org.broadinstitute.dsde.workbench.client.sam.ApiException;
+import org.broadinstitute.dsde.workbench.client.sam.api.AdminApi;
+import org.broadinstitute.dsde.workbench.client.sam.api.UsersApi;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SamService {
+  private final SamConfiguration samConfig;
+
+  @Autowired
+  public SamService(SamConfiguration samConfig) {
+    this.samConfig = samConfig;
+  }
+
+  private ApiClient getApiClient(String accessToken) {
+    ApiClient apiClient = getApiClient();
+    apiClient.setAccessToken(accessToken);
+    return apiClient;
+  }
+
+  private ApiClient getApiClient() {
+    return new ApiClient().setBasePath(samConfig.basePath());
+  }
+
+  public UsersApi samUsersApi(String accessToken) {
```

Comment:
not used?

Reply:
Thanks, forgot to clean it up after testing